pkg/apis/kosmos/v1alpha1/knode_types.go

```diff
@@ -28,6 +28,14 @@ type KnodeSpec struct {
 	// +optional
 	Kubeconfig []byte `json:"kubeconfig,omitempty"`
 
+	// +kubebuilder:default=50
+	// +optional
+	KubeAPIQPS float32 `json:"kubeAPIQPS,omitempty"`
```

Comment:
you can select or not select KubeAPIQPS

Reply:
> omitempty means the field should be omitted from the JSON output when encoding if it has an empty value (e.g., zero value for its type)”
from https://askgolang.com/omitempty-in-golang/

Reply:
learned